script/script.js

```diff
@@ -184,6 +187,11 @@ var Vector2 = (function() {
         ctx.stroke();
     };
 
+    // Send packet to set player's direction on a delay, to avoid running in to rate-limit.
+    setInterval(function() {
+        sendPacket(targetDirection);
+    }, 55);
+
```

Comment:
Perhaps there is a way of rate limiting without trading off reaction time - for example, have some angle-delta variable, where if the snake's current heading differs from `targetDirection` by at least `dA`, an update is sent. What do you think?